<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OwnerProfile</class>
 <widget class="QDialog" name="OwnerProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="generalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="profileTab">
        <attribute name="title">
         <string>Profile</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="profileLayout">
           <item>
            <layout class="QHBoxLayout" name="nicknameLayout">
             <item>
              <widget class="QLabel" name="nicknameLabel">
               <property name="text">
                <string>Your nickname:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nicknameEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="infoLayout">
             <item>
              <widget class="QLabel" name="infoLabel">
               <property name="minimumSize">
                <size>
                 <width>91</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Your info:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="infoEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="buttonLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="saveButton">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="passwordTab">
        <attribute name="title">
         <string>Password</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="passwordLayout" stretch="0,0,1,0">
           <item>
            <layout class="QHBoxLayout" name="curPasswordLayout">
             <item>
              <widget class="QLabel" name="curPasswordLabel">
               <property name="text">
                <string>Current password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="curPasswordEdit">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="newPasswordLayout">
             <item>
              <widget class="QLabel" name="newPasswordLabel">
               <property name="minimumSize">
                <size>
                 <width>110</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>New password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="newPasswordEdit">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="labelLayout">
             <item>
              <widget class="QLabel" name="infLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="buttonLayout_2">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="changeButton">
               <property name="text">
                <string>Change</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout_3" stretch="5,1">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
